docs/storage-management.asciidoc

```diff
@@ -3,73 +3,65 @@
+The amount of storage for APM data depends on several factors:
+how many services you are instrumenting, how much traffic the services see, agent and server settings,
+and for how long you keep monitoring data.
 
 [[reduce-sample-rate]]
 [float]
 === Reduce the sample rate
-In case you are monitoring high traffic services you might want to decrease the number of transactions that are sampled. 
-The transaction sample rate directly influences the number of documents to be indexed
-and therefore is the most obvious way to reduce storage. 
 
-The transaction sample rate is controlled in the configuration of agents (for example for {apm-py-ref}/configuration.html#config-transaction-sample-rate[Python] and {apm-node-ref}/agent-api.html#transaction-sample-rate[Node.js]).
+The transaction sample rate directly influences the number of documents (more precisely, Spans) to be indexed
+and therefore is the most obvious way to reduce storage.
 
-Reducing the transaction sample rate does not affect the collection of metrics such as _Transactions Per Second_.
+The transaction sample rate is a configuration of the agents.
+
+Reducing it does not affect the collection of metrics such as _Transactions Per Second_.
 
 [[reduce-stacktrace]]
 [float]
 === Reduce collected stacktrace information
-Elastic APM agents automatically collect information regarding stacktraces under certain circumstances. 
+Elastic APM agents collect `stacktrace` information under certain circumstances.
 This can be very helpful by identifying issues in your code,
-but it also comes with an overhead at collection time 
-and increaeses the storage usage. 
-If you want to decrease collected stacktrace information you have to configure this directly in the agents. 
-Please refer to our {apm-agents-ref}/index.html[agent documentation] to read more about the options in the single 
-agents. 
+but it also comes with an overhead at collection time and increases the storage usage.
+Stacktrace collection settings are managed in the agents.
 
 [[delete-data]]
 [float]
 === Delete data
-You might want to delete data for several reasons.
-A common use case is to keep data only for a defined time period and delete older documents. 
-You might also want to delete data collected for specific services or customers, 
-or delete specific indices. 
-Depending on your use case, 
-you can either delete data periodically with a tool like {curator-ref-current}[Curator] 
-or by using the {ref}/docs-delete-by-query.html[Delete By Query API]
-or by using the {kibana-ref}/managing-indices.html[Kibana Index Management UI]. 
+
+You might want to keep data only for a defined time period and delete old documents periodically,
+or you might also want to delete data collected for specific services or customers,
+or delete specific indices.
+Depending on your use case,
+you can either delete data periodically with a tool like {curator-ref-current}[Curator],
+using the {ref}/docs-delete-by-query.html[Delete By Query API]
```

Comment:
"by using" is clearer in both instances here, IMO, because you avoid the dangling modifier problem.